src/nvim/main.c

```diff
@@ -1841,10 +1841,10 @@ static void usage(void)
   mch_msg(_("  -p[N]                 Open N tab pages (default: one for each file)\n"));
   mch_msg(_("  +                     Start at end of file\n"));
   mch_msg(_("  +<linenum>            Start at line <linenum>\n"));
-  mch_msg(_("  +/<pattern>           Start at first occurrence of <pattern>\n"));
-  mch_msg(_("  --cmd <command>       Execute <command> before loading any nvimrc\n"));
-  mch_msg(_("  -c <command>          Execute <command> after loading the first file\n"));
-  mch_msg(_("  -S <session>          Source <session> after loading the first file\n"));
+  mch_msg(_("  +/<pattern>           Start at first occurrence of <pattern>\n"));  // NOLINT:whitespace
+  mch_msg(_("  --cmd <command>       Execute <command> before loading any vimrc\n"));  // NOLINT:whitespace
+  mch_msg(_("  -c <command>          Execute <command> after loading the first file\n"));  // NOLINT:whitespace
+  mch_msg(_("  -S <session>          Source <session> after loading the first file\n"));  // NOLINT:whitespace
```

Comment:
No need to clean this up ever--it has a good reason for not following style guide.